#
# Patch developed in house.
# The issue was reported to the Upstream by external person: Bug #3699.
#
--- openssl-0.9.8m/crypto/sparccpuid.S       Mon Jun 29 19:28:02 2015
+++ openssl-0.9.8m/crypto/sparccpuid.S       Mon Jun 29 19:28:02 2015
@@ -428,6 +428,8 @@
 .type	OPENSSL_cleanse,#function
 .size	OPENSSL_cleanse,.-OPENSSL_cleanse

+
+#if 0
 .global	_sparcv9_vis1_instrument_bus
 .weak	_sparcv9_vis1_instrument_bus
 .align	8
@@ -496,6 +498,8 @@
 .type	_sparcv9_vis1_instrument_bus2,#function
 .size	_sparcv9_vis1_instrument_bus2,.-_sparcv9_vis1_instrument_bus2

+#endif
+
 .section	".init",#alloc,#execinstr
 	call	solaris_locking_setup
 	nop
